Automattic/woocommerce-payments
# Fix and unskip shopper E2E tests

Fixes #3576 

#### Changes proposed in this Pull Request








- Fixes and unskips some of the failing shopper E2E tests.
- Change console suppression to loop through a list of messages to ignore.


#### Testing instructions





* Go to Actions->WCPay E2E tests -> Run workflow.
* Select this branch : `fix/unskip-shopper-e2e-tests`.
* Run the workflow and ensure all wcpay shopper tests pass, [reference job](https://github.com/Automattic/woocommerce-payments/actions/runs/1728864546).

-------------------


**Post merge**
